service/swagger.js

```diff
@@ -39,7 +39,8 @@ const Team = Joi.object({
   path: Joi.string(),
   users: Joi.array().items(UserRef),
   policies: Joi.array().items(PolicyRef),
-  organizationId: Joi.string()
+  organizationId: Joi.string(),
+  membersCount: Joi.number()
```

Comment:
we don't return count on the single team return. Thinking about it seems to me that we should implement count also for single get to have a unified approach. @p16 ?
usersCount is a better naming, i'll change it in the next PR